docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:xenial
+
+RUN apt-get update \
+    && apt-get install -y \
+        curl \
+        lsb-release \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
This should be done last (after all apt-get commands). I believe auto remove depends on these files to some extent.

Reply:
Sounds reasonable.

Reply:
Changed.